package exceptions.testthrows;

public class A {
	public static void f1(){
		B.f2();
	}
}
